Reports/progress_report.md

```diff
@@ -0,0 +1,296 @@
+# Data Safe Haven project report
```

Comment:
Ah yes definitly, it wa a confusing one because I wanted it to be for the monthly but in practice has everything since the last strategy session. Will add and then update.

Do we feel it is best to have one report that updates or monthly reports?

Reply:
In TPS we do a single file for each 6 month report that we add to each month and I think it works really well. Otherwise it can really feel like you're repeating a lot of information each month. So doing the monthly updates 1) lets you focus on a pull request on what has _changed_ rather than having to have two files open to compare, and 2) lets the project manager (@Davsarper!) build up the report so there's not much extra work to do by the time submitting the report comes around!